h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -1434,11 +1434,14 @@ def generate_for_indices(list_size, check_all, num_per_dim, start_val):
 
 def generate_sign_vec(table1, table2):
     sign_vec = [1]*len(table1.cell_values[0])
-    for indC in range(1, len(table2.cell_values[0])):
-        if (np.sign(table1.cell_values[0][indC])!=np.sign(table2.cell_values[0][indC])):
-            sign_vec[indC] = -1
-        else:
-            sign_vec[indC] = 1
+    for indC in range(1, len(table2.cell_values[0])):   # may need to look at other elements since some may be zero
+        for indR in range(0, len(table2.cell_values)):
+            if (abs(table1.cell_values[indR][indC]) > 0) and (abs(table2.cell_values[indR][indC]) > 0):
+                if (np.sign(table1.cell_values[indR][indC])!=np.sign(table2.cell_values[indR][indC])):
+                    sign_vec[indC] = -1
+                else:
+                    sign_vec[indC] = 1
```

Comment:
I believe you can reach the same result by
sign_vec[IndC] = (np.sign(table1.cell_values[indR][indC]) * np.sign(table2.cell_values[indR][indC])